contracts/LockedNORI.sol

```diff
@@ -244,31 +244,33 @@ contract LockedNORI is
   }
 
   /**
-   * @notice Unwrap BridgedPolygonNORI tokens and makes them available for use in the BridgedPolygonNORI contract
+   * @notice Claim unlocked tokens.
    *
-   * @dev This function burns `amount` of wrapped tokens and withdraws them to the corresponding {BridgedPolygonNORI}
-   * tokens.
+   * @dev This function burns `amount` of {LockedNORI} and transfers `amount`
```

Comment:
What's the reasoning behind using the `{contract-name}` syntax? Afaik this is used in oz repos for their custom docs tempgenrator, but maybe there's another reason to us it?

Reply:
Oh, yeah I was just trying to follow their doc styles but I guess it doesn't matter since our doc generator doesn't paint these as links to the other contracts.